pkg/k8s/client.go

```diff
@@ -571,5 +576,5 @@ func (c *Client) ListServiceExports() []*mcs.ServiceExport {
 
 // AddMeshRootCertificateEventHandler adds an event handler specific to mesh root certificiates.
 func (c *Client) AddMeshRootCertificateEventHandler(handler cache.ResourceEventHandler) {
-	c.informers[informerKeyMeshRootCertificate].AddEventHandler(handler)
+	c.informers[informerKeyMeshRootCertificate].AddEventHandlerWithResyncPeriod(handler, MRCResyncInterval)
```

Comment:
nit: i might add the resync interval as a param to the method, and let the caller decide.